tests/Integration/Provider/SentinelServiceProviderTest.php

```diff
@@ -11,14 +11,11 @@
  * @see https://github.com/opencfp/opencfp
  */
 
-namespace OpenCFP\Test\Unit\Provider;
+namespace OpenCFP\Test\Integration\Provider;
 
 use Cartalyst\Sentinel\Sentinel;
 use OpenCFP\Test\BaseTestCase;
 
-/**
- * @covers \OpenCFP\Provider\SentinelServiceProvider
- */
 final class SentinelServiceProviderTest extends BaseTestCase
 {
     public function testAllRepositoriesAreSet()
```

Comment:
Since we access the application container in this test, I believe this should be an integration test.